magellan-library/src/main/java/com/wealthfront/magellan/navigation/ActionBarConfigListener.java

```diff
@@ -3,9 +3,10 @@
 import com.wealthfront.magellan.view.ActionBarConfig;
 
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public interface ActionBarConfigListener {
 
-  void onNavigate(@NotNull ActionBarConfig actionBarConfig);
+  void onNavigate(@Nullable ActionBarConfig actionBarConfig);
```

Comment:
This method needs to be called for screen that actually done implement this interface as well. Steps don't automatically implement this interface.